XCUITests/AuthenticationTest.swift

```diff
@@ -105,6 +105,8 @@ class AuthenticationTest: BaseTestCase {
     }
 
     func testChangeRequirePasscodeInterval() {
+        navigator.performAction(Action.CloseURLBarOpen)
```

Comment:
oh just realized this changed was added as I tested that test :( I can remove it from this PR if you have it in yours

Reply:
Removed